.circleci/config.yml

```diff
@@ -591,54 +614,24 @@ jobs:
   ### e2e/integration test jobs ###
   #################################
 
-  e2e:
-    machine:
-      image: ubuntu-2004:202010-01
-    resource_class: large
-    environment:
-      GOCACHE: /tmp/go-cache
-    working_directory: /home/circleci/go/src/github.com/influxdata/influxdb
+  e2e-monitor-ci:
+    docker:
+      - image: cimg/base:2021.04
     steps:
-      - browser-tools/install-browser-tools
-      - checkout
-      - attach_workspace:
-          at: /home/circleci/go/src/github.com/influxdata/influxdb
-      - restore_cache:
-          name: Restore Yarn Cache
-          keys:
-            - yarn-deps-lock-{{ checksum "ui/yarn.lock" }}
-      - run: sudo apt-get update && sudo apt-get install netcat-openbsd
+      # if we are running from a fork, exit with a fail status.
+      - bail_if_forked
       - run:
-          name: Install Dependencies
-          command: |
-            set +e
-            cd ui
-            yarn install --immutable
-      - save_cache:
-          name: Save Yarn Cache
-          key: yarn-deps-lock-{{ checksum "ui/yarn.lock" }}
-          paths:
-            - /home/circleci/go/src/github.com/influxdata/influxdb/ui/node_modules
-            - ~/.cache/yarn
-            - ~/.cache/Cypress
-      - run:
-          name: Start the OSS docker image
-          command: |
-            docker load < docker-image.tar
-            docker run -p 8086:8086 \
-              -e INFLUXD_STORE=memory \
-              -e INFLUXD_E2E_TESTING=true \
-              quay.io/influxdb/oss-acceptance:${CIRCLE_SHA1}
-          background: true
-      - run: make e2e
-      - store_test_results:
-          path: ui/junit-results
+          name: Run the tests
+          command: wget -O - https://raw.githubusercontent.com/influxdata/ui/master/scripts/run-monitor-ci-tests.bash | API_KEY=${MONITOR_CI_API_KEY} MONITOR_CI_BRANCH="master" OSS_SHA=${CIRCLE_SHA1} bash
```

Comment:
I guess this is secure-ish since it's from a private GitHub repo, but `wget | bash` is how the Codecov security breach spread around so it makes me nervous.

How often does this script change in `monitor-ci`? Would it be a pain to keep a copy in-sync from our repo?

Reply:
Agreed that we should find a different way of doing this. We should at least be verifying the sha256 checksum of the script.

Reply:
Haha...incredible timing about the Codecov breach happening because of this. I think keeping a copy of the script in our own repo is probably the most practical way to do it right now.

Reply:
I'm including a copy of this script now in the `influxdb` repo as well. It should be easy enough to port over changes from the `ui` repo when it makes sense.